packages/bridge/src/contexts/ethereum.tsx

```diff
@@ -168,6 +171,7 @@ export const EthereumProvider: FunctionComponent = ({ children }) => {
       });
       // @ts-ignore
       wallet.on('accountsChanged', accounts => {
+        // @ts-ignore
```

Comment:
These `@ts-ignore` comments were added because `accountsChanged` is not a valid event emitted by the adapter interface, and the interface is being used to describe the incompatible WalletConnect and Metamask adapters specific to this package. The reason this compiled previously is because `wallet` was typed as `any` in most places.